src/ui/select-base/SelectBase.tsx

```diff
@@ -31,6 +31,7 @@ namespace SelectBase {
     dataIdx: string;
     label: string;
     value: boolean;
+    pictureUrl?: string;
```

Comment:
Remove pictureUrl. SelectBase don't care about other properties.

Reply:
Ok, it's been removed - therefore implementation of getting pictureUrl must have been changed. Check this out and give me a feedback if this approach is acceptable